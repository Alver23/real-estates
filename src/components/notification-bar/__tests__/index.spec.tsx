// Dependencies
import { shallow, ShallowWrapper } from 'enzyme';

// Under test file
import NotificationBar from '../index';

describe('<NotificationBar />', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<NotificationBar />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });
});
