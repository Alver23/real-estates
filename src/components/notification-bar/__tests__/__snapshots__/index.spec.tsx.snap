// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotificationBar /> should save a snapshot of the component 1`] = `
<div
  className="bg-green-300 w-3 h-3 rounded-full"
/>
`;
